lib/rack/exceptions.rb

```diff
@@ -1,26 +1,36 @@
 module Rack
   class Exceptions
+    # XXX: applications are typically configured with a single global instance
```

Comment:
Looks legit, but why the `XXX:`?

Reply:
i felt gross doing this, i thought there could've been a way around it by passing references the NORMAL way. but if i think about it more, i think it's a necessary evil b/c we want Exceptions to be global. removing the XXX haha